packages/owl-bot/src/bin/commands/commit-post-processor-update.ts

```diff
@@ -181,3 +195,38 @@ export function commitOwlbotUpdate(repoDir: string) {
   console.log(`git commit -m "${commitMessage}"`);
   proc.spawnSync('git', ['commit', '-m', commitMessage], {cwd: repoDir});
 }
+
+async function promoteFromDraft(
+  prLocator: PRLocator,
+  githubToken: string
+): Promise<void> {
+  const octokitFactory = octokitFactoryFromToken(githubToken);
+  const octokit = await octokitFactory.getShortLivedOctokit();
+  const found = (await octokit.graphql(
+    `
+    query findPullRequestID($owner: String!, $repo: String!, $pullNumber: Int!) {
+      repository(owner:$owner, name:$repo) {
+        pullRequest(number:$pullNumber) {
+          id
+        }
+      }
+    }`,
+    {
+      owner: prLocator.owner,
+      repo: prLocator.repo,
+      pullNumber: prLocator.pull_number,
+    }
+  )) as any;  // eslint-disable-line
+
+  await octokit.graphql(
```

Comment:
Should we check the result of the first query in case we don't get the expected response?